webapp/src/components/social/UserPlaceHolder.tsx

```diff
@@ -45,26 +45,42 @@ export default class UserPlaceHolder extends React.Component<UserPlaceHolderProp
         });
     }
 
+    private shortText(text: string | null) {
+        if (text === null) {
+            return "Unknwon friend";
```

Comment:
"Unknown"

Reply:
I will change it right away after the merge!